pithos/pithos.py

```diff
@@ -520,6 +520,7 @@ def user_pause(self, *ignore):
 
     def pause(self):
         self.playing = False
+        self.destroy_ui_loop()
```

Comment:
This call is unnecessary. Once update_song_row returns False (aka paused) it will already destroy it.